plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1191,6 +1197,9 @@ public String toString() {
             if (_networkName != null) {
                 netBuilder.append("<target dev='" + _networkName + "'/>\n");
             }
+            if (NetUtils.isValidMtu(_mtu)) {
```

Comment:
As we check it here we can just pass a null when the MTU is not set. No need to hardcode 1500 in the code.